test/helpers/kubectl.go

```diff
@@ -808,9 +806,20 @@ func (kub *Kubectl) NamespaceCreate(name string) *CmdRes {
 
 // NamespaceDelete deletes a given Kubernetes namespace
 func (kub *Kubectl) NamespaceDelete(name string) *CmdRes {
+	res := kub.DeleteAllInNamespace(name)
+	if !res.WasSuccessful() {
+		kub.Logger().Infof("Error while deleting all objects from %s ns: %s", name, res.GetError().Error())
+		return res
```

Comment:
Maybe we shouldn't return early in this case? deleting the namespace is supposed to work... but it sometimes doesn't so this step is sort of extra.